src/pages/iou/request/step/IOURequestStepDistance.js

```diff
@@ -38,19 +40,24 @@ const propTypes = {
 
     /** The transaction object being modified in Onyx */
     transaction: transactionPropTypes,
+
+    /** backup version of the original transaction  */
+    transactionBackup: transactionPropTypes,
```

Comment:
I'm not feeling confident about this part... Is it a conventional way to handle transactions in Expensify? I don't think I've ever seen transaction "backups".

Reply:
@cubuspl42 This change has been used before. I am migrating logic to new file

https://github.com/Expensify/App/blob/d98ee75de6ba8dff26c8261a63d50d7e034cb741/src/pages/EditRequestDistancePage.js#L71

Reply:
Okey, I'll take a look at the old code. Thanks!